### PR TITLE
sd-device: store parsed SEQNUM and ACTION string

### PR DESCRIPTION
**Pull request by yuwata**
When a device is passed through netlink uevent, then we parsed `ACTION=` and `SEQNUM=` property to check the validity of the message, but the parsed result was discarded. And these values were re-parsed when needed.

This makes the parsed values stored in `sd_device`, so re-parsing these properties is not necessary any more.

### CODE REVIEW

File: src/shared/udev-util.c [hunk 1]
```diff
@@ -181,3 +181,14 @@ int device_is_renaming(sd_device *dev) {
 
         return r >= 0;
 }
+
+bool device_is_in_action(sd_device *dev, DeviceAction action) {
```
Comment (poettering) [inline src/shared/udev-util.c]: maybe device_for_action()?

File: src/libudev/libudev-device.c [hunk 2]
```diff
@@ -45,24 +45,15 @@
...
                 return 0;
-        else if (r < 0)
-                return_with_errno(0, r);
-
-        r = safe_atollu(seqnum, &ret);
-        if (r < 0)
-                return_with_errno(0, r);
 
-        return ret;
+        return (unsigned long long) seqnum;
```
Comment (poettering) [inline src/libudev/libudev-device.c]: i figure an implicit cast is good enough here...

**Comment by poettering** [comment]
looks good to me, two minor nits above

**Comment by yuwata** [comment]
@poettering Thank you for your comments. I've force-pushed the revised version.
- Function name is changed from `device_is_in_action()` to `device_for_action()`.
- Drop explicit cast to `unsigned long long`.

**Comment by poettering** [comment]
hmm, the ci failure looks unrelated, but I am not sure?

**Comment by poettering** [comment]
This looks good to me. But could you push a rebased version maybe to restart the CI? would love to see the Ci pass first. (or is there another way to restart the centos ci?)

**Comment by yuwata** [comment]
Rebased.

**Comment by poettering** [comment]
hmm, centos ci failed again on this one?

**Comment by yuwata** [comment]
Re-rebased, and added a workaround for the failed test.

**Comment by yuwata** [comment]
Hmm, ubuntu CIs are broken now...

File: test/TEST-29-UDEV-ID_RENAMING/testsuite.sh [hunk 3]
```diff
@@ -21,19 +21,19 @@ EOF
 udevadm control --log-priority=debug --reload
 udevadm trigger --action=add --settle /sys/devices/virtual/net/lo
 udevadm info /sys/devices/virtual/net/lo
-
+sleep 1
```
Comment (keszybz) [inline test/TEST-29-UDEV-ID_RENAMING/testsuite.sh:24]: IIUC, `udevadm --settle` waits for udev, but it does not wait for systemd, which is asynchronous, so we can get wrong state from systemctl. Sleeping here seems a reasonable workaround.

Comment (yuwata) [reply test/TEST-29-UDEV-ID_RENAMING/testsuite.sh:24]: Yeah, I think so.

**Pull request merged**